src/peft/optimizers/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023-present the HuggingFace Inc. team.
```

Comment:
```suggestion
# Copyright 2024-present the HuggingFace Inc. team.
```

Reply:
Sure thing. But isn't the left part of copyright years normally kept as whatever year it began to exist?